sql/updates/latest-dev.sql

```diff
@@ -1,2 +1,162 @@
 DROP VIEW IF EXISTS timescaledb_information.continuous_aggregates;
 DROP VIEW IF EXISTS timescaledb_information.job_stats;
+
+--Modify hypertable table.
+DROP VIEW IF EXISTS timescaledb_information.hypertables;
+DROP VIEW IF EXISTS timescaledb_information.chunks;
+
+--alter compressed column type
+ALTER TABLE IF EXISTS _timescaledb_catalog.hypertable 
```

Comment:
I would caution against modifying a table too much in place, as opposed to recreating it completely. We often see table indexes get messed up this way. Do we know for sure that, e.g., changing a columns type doesn't really do a drop/add column internally?

Reply:
From PG docs : https://www.postgresql.org/docs/current/sql-altertable.html
``` Adding a column with a volatile DEFAULT or changing the type of an existing column will require the entire table and its indexes to be rewritten. As an exception, when changing the type of an existing column, if the USING clause does not change the column contents and the old type is either binary coercible to the new type or an unconstrained domain over the new type, a table rewrite is not needed; but any indexes on the affected columns must still be rebuilt. ```

Reply:
This sounds to me as if the modified column will actually not be strictly the same as a cleanly and newly created one. The column will stay with the old type but existing values converted to the new type. Is this really what we want? It seems unnecessarily risky to me.

How will this affect future update scripts, for instance, when the column definition can be different depending on whether the DB was previously upgraded or created fresh.

Reply:
might be a good idea to run cluster after the operation so it gets rewritten